### DISCUSSION TITLE
How can I cancel and get a refund for my GitHub Enterprise Cloud annual subscription?

### DISCUSSION THREAD

**Discussion by yoyozo** [community/community#143068] [Enterprise] [answered]
### Select Topic Area

Question

### Body

How can I cancel and get a refund for my GitHub Enterprise Cloud annual subscription?

**Comment by queenofcorgis** [comment, marked as answer]
@yoyozo have you checked out [this doc](https://docs.github.com/en/enterprise-cloud@latest/billing/managing-the-plan-for-your-github-account/downgrading-your-accounts-plan#downgrading-your-enterprise-accounts-plan) on downgrading your plan? Let me know if you still have questions!

**Reply by yoyozo** [reply]
How can I check the processing results and schedule? Even after deleting my enterprise account, the billing charges have not been canceled. How can I address this

**Reply by queenofcorgis** [reply]
I recommend referring to our [customer terms page](https://github.com/customer-terms) for policies around refunds.
